src/api/v2/v2AzureResourcesApiWrapper.ts

```diff
@@ -17,6 +17,14 @@ export class V2AzureResourcesApiWrapper implements V2AzureResourcesApi {
         return this.api.apiVersion;
     }
 
+    get applicationResourceTreeDataProvider(): vscode.TreeDataProvider<unknown> {
+        return this.callWithTelemetryAndErrorHandlingSync('v2.getApplicationResourceTreeDataProvider', () => this.api.applicationResourceTreeDataProvider);
```

Comment:
Does a simple `get` method really deserve telemetry wrapping?

Reply:
Ya... it's probably overkill